flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/PartitionRequestClientHandler.java

```diff
@@ -292,7 +296,11 @@ else if (bufferListener.waitForBuffer(bufferProvider, bufferOrEvent)) {
 						return false;
 					}
 					else if (bufferProvider.isDestroyed()) {
-						return false;
```

Comment:
We usually have a white space between keywords like `if` or `else`:
```java
if (isStagedBuffer) {
    return true;
} else {
    return false;
}
```

In this case, you can simplify the return value to `return isStagedBuffer`. Same for the other place where you use this.